### PR TITLE
Functional iOS/Android test

### PR DESCRIPTION
**Pull request by grozen**
This is the very first integration test that works for both our iOS and Android example apps.
All it does is register and perform a purchase.

Do have a look while I try to get this to run on Travis.

### CODE REVIEW

File: features/step_definitions/typical_steps.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+When(/^I launch the sample application and press '(.+)'$/) do |button_text|
+  button(button_text).click
```
Comment (tomerlevi444) [inline features/step_definitions/typical_steps.rb:7]: I think that "I launch the sample application" is not necessary.

Comment (yuval-netanel) [inline features/step_definitions/typical_steps.rb:7]: +1

**Comment by yuval-netanel** [comment]
Missing read.me on how to run/install it

**Comment by yuval-netanel** [comment]
Amazing!

**Comment by grozen** [comment]
@yuval-netanel You are correct that this is in desperate need of a README, and I will add one in a later pull request.

**Comment by tomerlevi444** [comment]
Approved

**Pull request merged**